ModelicaTest/Electrical/QuasiStatic/Polyphase.mo

```diff
@@ -226,5 +226,6 @@ package Polyphase "Polyphase quasi-static package"
     connect(starI.plug_p,currentSource. plug_p) annotation (Line(points={{-20,-80},{-20,-50}}, color={85,170,255}));
     connect(currentSource.plug_p, capacitor.plug_n) annotation (Line(points={{-20,-50},{-10,-50},{-10,-20},{-20,-20}}, color={85,170,255}));
     connect(capacitor.plug_p,currentSource. plug_n) annotation (Line(points={{-40,-20},{-50,-20},{-50,-50},{-40,-50}}, color={85,170,255}));
+  annotation(experiment(StopTime = 1.0, Interval = 0.0002));
```

Comment:
```suggestion
  annotation(experiment(StopTime = 1.0));
```
I don't see a need for Interval, as the other examples in the same package doesn't have it and still has the same number of outputs, and there is nothing interesting in these signals at that time-scale. I guess the added points are added to make the references "better" in some way.

Reply:
Seems reasonable to me. @christiankral, @AHaumer do you agree?

Reply:
Since a ramp is used, I would use Interval in the range 0.01 ... 0.001

Reply:
Are we discussing the same model? I don't see a ramp in this one (ModelicaTest.Electrical.QuasiStatic.Polyphase.FrequencySweep), but in the other two examples in the package that already had experiment-annotation.

Reply:
(without Interval for the other two).